backends/arm/operators/op_permute.py

```diff
@@ -40,8 +88,33 @@ def define_node(
             )
             return
 
+        # The permutation vector describes a permutation P in default Pytorch dim_order.
+        # For rank 4, the default dim_order NCHW.
+        # E.g. (2,3,0,1) -> permute (n,c,h,w) to (w,c,n,h)
```

Comment:
```suggestion
        # E.g. (2,3,0,1) -> permute (n,c,h,w) to (h,w,n,c)
```

Reply:
Your suggestion is correct, messed this up.